store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -115,13 +123,19 @@ class FlowStoreTest {
             )
         pipeline.stream(StoreRequest.cached(3, refresh = false))
             .assertItems(
+                testScope,
                 Data(
                     value = "three-1",
                     origin = Cache
+                ),
```

Comment:
FYI this is not a bug.
we do not observe the persister for every item in the cache (unless there is an active stream for it).
this is why we need to get the disk value again.

Reply:
Could be good as a comment for people that learn functionality through tests